### PR TITLE
Update names to PR changelog

### PR DESCRIPTION
**Pull request by ankita240796**
## Explanation
Change label names to PR CHANGELOG

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by ankita240796** [comment]
Hi @kevinlee12, can you please ping me before merging this. I will change the labels simultaneously  to avoid any extra comments from the bot. Thanks!

File: lib/checkPullRequestLabels.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ module.exports.checkAssignee = async function(context) {
     pullRequestNumber + ' ...');
 
   var changelogLabel = context.payload.label.name.trim();
-  var hasChangelogLabel = changelogLabel.toUpperCase().startsWith('CHANGELOG');
+  var hasChangelogLabel = changelogLabel.toUpperCase().startsWith(
+    'PR CHANGELOG');
```
Comment (kevinlee12) [inline lib/checkPullRequestLabels.js]: Let's rename this to `PR: CHANGELOG` to be consistent with other PR labels.

File: lib/checkPullRequestLabels.js [hunk 2]
```diff
@@ -54,7 +55,7 @@ module.exports.checkChangelogLabel = async function(context) {
   var userName = pullRequest.user.login;
   var changelogLabel = '';
   var hasChangelogLabel = labels.some(function(label) {
-    if (label.name.trim().toUpperCase().startsWith('CHANGELOG')) {
+    if (label.name.trim().toUpperCase().startsWith('PR CHANGELOG')) {
```
Comment (kevinlee12) [inline lib/checkPullRequestLabels.js]: Ditto

File: lib/checkPullRequestLabels.js (hunk 1 above)
Comment (kevinlee12) [reply lib/checkPullRequestLabels.js]: ^ mind the colon

Comment (ankita240796) [reply lib/checkPullRequestLabels.js]: Replied here: https://github.com/oppia/oppia/pull/8515#discussion_r372218867

File: lib/checkPullRequestLabels.js (hunk 2 above)
Comment (ankita240796) [reply lib/checkPullRequestLabels.js]: Replied above.

**Comment by ankita240796** [comment]
Hi @kevinlee12, PTAL. Thanks!

**Comment by kevinlee12** [comment]
@ankita240796 I'll let you merge this, deployment does occur manually though (so a merge to master will not auto deploy).

**Comment by ankita240796** [comment]
Hi @kevinlee12, Can you merge this in that case? I will update the labels whenever you deploy. Thanks!

**Pull request merged**